build.gradle

```diff
@@ -54,6 +55,25 @@ java {
 
 javadoc {
     options.encoding = 'UTF-8'
+    options.windowTitle = 'jspecify 0.0.0-SNAPSHOT API'
```

Comment:
As best I can tell, this is unnecessary: Gradle fills it in. (I did have to pass it manually when I was running javac directly.)

Now, it sounds like today's Javadoc defaults might not require the `encoding` line above, either. But at least that one won't require a single-bit edit as part of the 1.0.0 release :)